docs/installation.md

```diff
@@ -89,36 +103,44 @@ OS Specific steps are listed first, the [Common](#common) section below is neces
 !!! Note
     Python3.6 or higher and the corresponding pip are assumed to be available.
 
-### Linux - Ubuntu 16.04
+=== "Ubuntu 16.04"
```

Comment:
How wold we expect this to render / are you sure this renders correctly in the readthedocs context (if yes - got any samples on how this looks like when deployed to readthedocs?)

Reply:
This renders perfectly fine:

Sample1:
![image](https://user-images.githubusercontent.com/37382997/92131389-2bf36b00-edc3-11ea-8ed6-7f0c60a007e8.png)

Sample2:
![image](https://user-images.githubusercontent.com/37382997/92131493-4c232a00-edc3-11ea-902f-406f8f5d8c7e.png)


The goal is to provide Platform specific instructions